website/src/components/IconButton/IconButton.test.jsx

```diff
@@ -20,7 +20,39 @@ describe('IconButton', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
-  it('should have a class name called IconButton', () => {
-    expect(shallow(<IconButton type="setting" onClick={sampleFunction} />).exists('.IconButton')).toBe(true);
+  it('should have a type called settings', () => {
+    const sampleFunctionfn = jest.fn(sampleFunction);
```

Comment:
Nitpick on the variable name `sampleFunctionfn` sounds wierd `sampleFunction` would probably be fine

Reply:
Oh that is because that function is mock, I used jest.fn() thereffore, I named it sampleFuntionFn